README.md

```diff
@@ -3,8 +3,43 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. And after a year and a half, we're making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
+> Empower people to sustainably unlock their potential and pursue their passion through technology.
```

Comment:
Sounds like we're in agreement - not sure it needed all of that? Bare in mind this is the same point I made before I joined and in my first week - I've actually mentioned it a number of times before so I'm surprised by the reaction here.